controls/RHEL-07-040190.rb

```diff
@@ -31,8 +31,8 @@
 If “ClientAliveInterval” is not set to “600” in /etc/ ssh/sshd_config, and a lower value is not documented with the Information System Security Officer (ISSO) as an operational requirement, this is a finding.'
 
 # START_DESCRIBE RHEL-07-040190
-  describe file('') do
-    it { should match // }
+  describe sshd_config do
+    its('ClientAliveInterval') { should eq '600' }
```

Comment:
I think the value we are checking against should be an attribute. By default they want 600 seconds, but it can also be set to a different value with a documentation. If I set it higher or lower, this check will still fail